wp-content/mu-plugins/https.php

```diff
@@ -25,25 +26,27 @@ function wp_calculate_image_srcset( $sources ) {
 
 /**
  * Replace http:// with https:// in the embed code (before caching).
- * 
+ *
+ * @since  2.0.0
```

Comment:
I was on a "minor" road, then I remembered that and went to the site which put me on a "major" eheh. It's just because of the "API" thing, but concede that it's a bit far fetched in a way 😄 .

Let the other speak then... others?